vm/inc/ubpf.h

```diff
@@ -102,6 +102,13 @@ int ubpf_exec(const struct ubpf_vm *vm, void *mem, size_t mem_len, uint64_t* bpf
 
 ubpf_jit_fn ubpf_compile(struct ubpf_vm *vm, char **errmsg);
 
+/*
+ * A wrapper around some of the compile, load and run commands to
+ * enable instruction counting. Returns the return value of the eBPF
```

Comment:
Probably don't need to document this function as "enable instruction counting" it's a helper to run the vm with or without-jit.... The fact that you are doing this to enable instruction counting is important in the commit message but not in the API documentation.